Assets/Scenes/Levels/Level1.unity

```diff
@@ -38,7 +38,7 @@ RenderSettings:
   m_ReflectionIntensity: 1
   m_CustomReflection: {fileID: 0}
   m_Sun: {fileID: 170076734}
-  m_IndirectSpecularColor: {r: 0.17487687, g: 0.21875805, b: 0.30104116, a: 1}
+  m_IndirectSpecularColor: {r: 0.1748765, g: 0.21875761, b: 0.30104014, a: 1}
```

Comment:
Please don't modify levels in this PR, it's not the part of the menu.

Reply:
Restored to your file